engine/dlib/src/dlib/lz4.cpp

```diff
@@ -36,27 +35,6 @@ namespace dmLZ4
             *decompressed_size = -1;
             r = dmLZ4::RESULT_OUTPUT_SIZE_TOO_LARGE;
         }
-
```

Comment:
Removed this function since:
* LZ4_decompress_fast was deprecated in favor of LZ4_decompress_safe
* LZ4_decompress_safe is faster (according to lz4 documentation) than LZ4_decompress_fast* functions
* LZ4_decompress_safe respects the buffer bounds